### PR TITLE
Update benchmark

### PR DESCRIPTION
**Pull request by ThorSedeke**
Update bench.js from matcha to benchmark.js couse of critical issues and stopped supported on matcha module.

### CODE REVIEW

File: bench.js [hunk 1]
```diff
@@ -48,9 +50,8 @@ bench('get', () => {
 	m.get('foo', 'foo.bar', false);
 	m.get([], 'foo.bar', false);
 	m.get(undefined, 'foo.bar', false);
-});
-
-bench('set', () => {
+})
+.add('set', function () {
```
Comment (sindresorhus) [inline bench.js]: I prefer keeping the arrow functions.

File: package.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
 		"node": ">=4"
 	},
 	"scripts": {
-		"test": "xo && ava",
-		"bench": "matcha bench.js"
+		"test": "xo index.js && ava",
```
Comment (sindresorhus) [inline package.json]: XO should test all JS files.

File: bench.js (hunk 1 above)
Comment (ThorSedeke) [reply bench.js]: Ok will change

**Comment by ThorSedeke** [comment]
Looks like xo has issues on node 4 
/home/travis/build/sindresorhus/dot-prop/node_modules/xo/main.js:115
const {input, flags: opts} = cli;

**Comment by ThorSedeke** [comment]
Installing xo on node 4
xo@0.21.1: wanted: {"node":">=6"} (current: {"node":"4.2.0","npm":"2.14.7"})

**Comment by sindresorhus** [comment]
Thanks :)

File: package.json [hunk 3]
```diff
@@ -39,7 +39,8 @@
 	},
 	"devDependencies": {
 		"ava": "*",
-		"matcha": "^0.7.0",
+		"benchmark": "^2.1.4",
+		"microtime": "^2.1.8",
```
Comment (stevemao) [inline package.json]: is this used somewhere?

**Pull request merged**